docs/source/flow-project.rst

```diff
@@ -92,6 +90,11 @@ A very simple *operation*, which creates a file called ``hello.txt`` within a jo
     if __name__ == "__main__":
         MyProject().main()
 
+.. tip::
+
+    By default operations only act on a single job and can simply be defined with the signature `def op(job)`.
+    When using :ref:`aggregate operations <aggregation>`, it is recommended to define operations as accepting a variadic list of ``*jobs`` parameters so that the operation is not restricted to a specific aggregate size.
```

Comment:
Need to define variadic

Reply:
"Variadic" is the right word to use, it's a term used in many programming languages. I think it's also self-explanatory because it's pointing clearly to the `*jobs` argument. It might be a little awkward to link to a blog post explaining how `*args` work but we could do something along those lines. @cbkerr How would you resolve this?

Reply:
```suggestion
    When using :ref:`aggregate operations <aggregation>`, it is recommended to allow the operation to accept a variable number of jobs using a variadic argument ``*jobs``, so that the operation is not restricted to a specific aggregate size.
```